AzureADSample/AzureADSample/View/MainPage.xaml

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="AzureADSample.MainPage">
+
+    <ScrollView>
+        <VerticalStackLayout 
+            Spacing="25" 
+            Padding="30,0" 
+            VerticalOptions="Center">
+
+            <Image
+                Source="dotnet_bot.png"
+                SemanticProperties.Description="Cute dot net bot waving hi to you!"
+                HeightRequest="200"
+                HorizontalOptions="Center" />
+
+            <Label 
+                Text="Azure AD Authentication Sample"
+                SemanticProperties.Description="Azure AD Authentication Sample"
+                FontSize="32"
+                HorizontalOptions="Center" />
```

Comment:
Try `HorizontalTextAlignment` here instead

```suggestion
                HorizontalTextAlignment="Center" />
```

Reply:
Got it!